.devcontainer/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.209.6/containers/javascript-node/.devcontainer/base.Dockerfile
 
 # [Choice] Node.js version (use -bullseye variants on local arm64/Apple Silicon): 16, 14, 12, 16-bullseye, 14-bullseye, 12-bullseye, 16-buster, 14-buster, 12-buster
-ARG VARIANT="16-bullseye"
+ARG VARIANT="24-bullseye"
```

Comment:
wonder if this should be changed to another debian version, for instance debian 13 named `trixie`:

https://www.debian.org/releases/

<img width="1217" height="339" alt="image" src="https://github.com/user-attachments/assets/33c9cbbe-6c9c-43fa-998f-34a61666d7b2" />